contracts/governance/airdrop.cairo

```diff
@@ -0,0 +1,2463 @@
+%lang starknet
+
+@storage_var
+func airdrop_claimed(claimee: Address) -> (res: felt) {
+}
+
+@external
+func claim{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(claimee: Address) {
+    alloc_locals;
+
+    let (governance_token_addr) = governance_token_address.read();
+
+    let claimed_already = airdrop_claimed.read(claimee);
+    with_attr error_message("claimee claimed already") {
+        assert claimed_already = 0;
+    }
+    let amt = get_eligible_amount(claimee);
+    let amt_u = Uint256(amt, 0);
+    claimed_already.write(claimee, amt);
+    IGovernanceToken.mint(contract_address=governance_token_addr, to=claimee, amount=amt_u);
+
+    return ();
+}
+
+func get_eligible_amount{range_check_ptr}(address: Address) -> felt {
+    if (address == 0x102fe99c69abdb8f30d1c8e6cbecd7224946ebbf964f5bca79f75129f44f014) {
```

Comment:
I think its gonna be ok, but could you double check/test that `0x102fe99c69abdb8f30d1c8e6cbecd7224946ebbf964f5bca79f75129f44f014` works as well as `0x0102fe99c69abdb8f30d1c8e6cbecd7224946ebbf964f5bca79f75129f44f014`... as address input :)

Reply:
@DaveVodrazka will need to test